src/Microsoft.ML.Core/Data/IHostEnvironment.cs

```diff
@@ -66,6 +66,11 @@ public interface IHostEnvironment : IChannelProvider, IProgressChannelProvider
         /// The catalog of loadable components (<see cref="LoadableClassAttribute"/>) that are available in this host.
         /// </summary>
         ComponentCatalog ComponentCatalog { get; }
+
+        /// <summary>
+        /// The seed property that, if assigned, makes components requiring randomness behave deterministically.
+        /// </summary>
+        int? Seed { get; }
```

Comment:
> @eerhardt commented [2020-02-04T16:32:54Z](https://github.com/dotnet/machinelearning/pull/4775#issuecomment-581996250)
> ...
> This is because you are adding a member to an interface, which is a breaking change. Interfaces, once shipped, can't be changed.

No need to expose a getter on the seed value. It's nice to be able to get it back, but the user sets the value and should have the value they gave.

> Should `HostBase` use the same `TauswortheHybrid` as the parent environment or create another one using the parent `_rand` as seed?

I assume `TauswortheHybrid` is not thread safe. So sharing can be an issue. Note: I haven't read the code for `HostBase`, so don't know the background.